### DISCUSSION TITLE
Flask+HTML autocomplete extensions for VS Code

### DISCUSSION THREAD

**Discussion by IArnaut2** [pallets/flask#5612] [Q&A] [answered]
Greetings,
Are there VS Code extensions that enable autocomplete for Flask+HTML? For example, when typing `{%` it turns into `{% %}`.

**Comment by CheeseCake87** [comment]
Is this what you're looking for?

https://marketplace.visualstudio.com/items?itemName=sugatoray.jinja-extension-pack

**Reply by IArnaut2** [reply, marked as answer]
I've installed better jinja, but thanks anyway.

**Reply by CheeseCake87** [reply]
No worries :) - I wasn't too sure, I use PyCharm myself...
